Description: Fix format errors in manpages
 This patch fixes minor format errors in several manpages.
Author: Manuel Prinz <manuel@debian.org>
Forwarded: https://svn.open-mpi.org/trac/ompi/ticket/2496
Applied-Upstream: https://svn.open-mpi.org/trac/ompi/changeset/23445
Last-Update: 2010-07-30

--- a/ompi/mca/crcp/ompi_crcp.7in
+++ b/ompi/mca/crcp/ompi_crcp.7in
@@ -8,8 +8,8 @@
 .\" **************************
 .SH NAME
 .
-Open MPI MCA Checkpoint/Restart Coordination Protocol (CRCP) Framework \- Overview of Open MPI's CRCP 
-framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
+OMPI_CRCP \- Open MPI MCA Checkpoint/Restart Coordination Protocol (CRCP) Framework:
+Overview of Open MPI's CRCP framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
 .
 .\" **************************
 .\"    Description Section
--- a/opal/mca/crs/opal_crs.7in
+++ b/opal/mca/crs/opal_crs.7in
@@ -14,8 +14,8 @@
 .\" **************************
 .SH NAME
 .
-Open PAL MCA Checkpoint/Restart Service (CRS) \- Overview of Open PAL's CRS 
-framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#. 
+OPAL_CRS \- Open PAL MCA Checkpoint/Restart Service (CRS): Overview of Open PAL's
+CRS framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#.
 .
 .\" **************************
 .\"    Description Section
--- a/orte/mca/filem/orte_filem.7in
+++ b/orte/mca/filem/orte_filem.7in
@@ -13,8 +13,8 @@
 .\" **************************
 .SH NAME
 .
-Open RTE MCA File Management (FileM) Framework \- Overview of Open RTE's FileM 
-framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
+ORTE_FILEM \- Open RTE MCA File Management (FileM) Framework: Overview of Open RTE's
+FileM framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
 .
 .\" **************************
 .\"    Description Section
--- a/orte/mca/snapc/orte_snapc.7in
+++ b/orte/mca/snapc/orte_snapc.7in
@@ -13,8 +13,8 @@
 .\" **************************
 .SH NAME
 .
-Open RTE MCA Snapshot Coordination (SnapC) Framework \- Overview of Open RTE's SnapC 
-framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
+ORTE_SNAPC \- Open RTE MCA Snapshot Coordination (SnapC) Framework: Overview of
+Open RTE's SnapC framework, and selected modules.  #PACKAGE_NAME# #PACKAGE_VERSION#
 .
 .\" **************************
 .\"    Description Section
--- a/orte/util/hostfile/orte_hosts.7in
+++ b/orte/util/hostfile/orte_hosts.7in
@@ -11,7 +11,8 @@
 .\" **************************
 .SH NAME
 .
-OpenRTE Hostfile and HOST Behavior \- Overview of OpenRTE's support for user-supplied hostfiles and comma-delimited lists of hosts
+ORTE_HOSTS \- OpenRTE Hostfile and HOST Behavior: Overview of OpenRTE's support for user-supplied
+hostfiles and comma-delimited lists of hosts
 .
 .\" **************************
 .\"    Description Section
